src/hooks/useVenueId.ts

```diff
@@ -9,7 +9,9 @@ const venuePaths = [
   "/e/:step/:venueId",
   "/in/:venueId",
   "/v/:venueId",
-  "/admin/venue/:venueId",
+  "/admin/:venueId",
+  "/admin/venue/edit/:venueId",
+  "/admin/venue/rooms/:venueId",
```

Comment:
[Potentially OOS] I backlogged https://github.com/sparkletown/internal-sparkle-issues/issues/164 in the past, about how we should be able to get rid of this 'paths' hack entirely.

Essentially, if we use `useParams`, it will be able to extract `:venueId` no matter where it exists in the url path:

- https://reactrouter.com/web/api/Hooks/useparams

Which I believe might even reduce this hook down to basically a single line.

Reply:
I agree with this but lets fix it in a separate PR.

Reply:
Sure

Reply:
FYI: I believe https://github.com/sparkletown/internal-sparkle-issues/issues/194#issuecomment-765080430 is related to the change I requested here.

Reply:
see https://github.com/sparkletown/sparkle/pull/1091 for fix